docs/develop/automatically-open-a-task-pane-with-a-document.md

```diff
@@ -108,6 +108,7 @@ The `webextension` part also includes a reference to the store or catalog with a
 | **`storeType` value** | **`id` value**    |**`store` value** | **`version` value**|
 |:---------------|:---------------|:---------------|:---------------|
 |OMEX (AppSource)|The AppSource asset ID of the add-in (see Note)|The locale of AppSource; for example, "en-us".|The version in the AppSource catalog (see Note)|
+|WOPICatalog (third-party [WOPI](https://wopi.readthedocs.io/en/latest/overview.html) hosts)| The AppSource asset ID of the add-in (see Note) | "wopicatalog". Use this value for add-ins that are published in App Source and are installed in third-party WOPI hosts. For more information, see [Enable Office Web Add-Ins provided by Host](https://wopi.readthedocs.io/en/latest/scenarios/hostinstalladdins.html) | The version in the add-in manifest.|
```

Comment:
It looks like these pages now redirect to docs so can you update to final destinations? However, the second link is also redirecting to the overview page so I'm not sure what the actual link should be.
```suggestion
|WOPICatalog (third-party [WOPI](/microsoft-365/cloud-storage-partner-program/online/) hosts)| The AppSource asset ID of the add-in (see Note). | "wopicatalog". Use this value for add-ins that are published in App Source and are installed in third-party WOPI hosts. For more information, see [Enable Office Web Add-Ins provided by Host](/microsoft-365/cloud-storage-partner-program/online/). | The version in the add-in manifest.|
```

Reply:
Thanks. I'll investigate the links and let you know.